.prettierrc

```
@@ -0,0 +1,17 @@
+{
+  "tabWidth": 2,
+  "useTabs": false,
+  "printWidth": 80,
+  "semi": true,
+  "singleQuote": true,
+  "quoteProps": "as-needed",
+  "jsxSingleQuote": false,
+  "trailingComma": "none",
```

trailing comma가 있으면 커밋 간 diff를 확인할 때 훨씬 효율적입니다. MDN의 [Trailing Commas](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas)에 나온 내용 첨부합니다

_Trailing commas (sometimes called "final commas") can be useful when adding new elements, parameters, or properties to JavaScript code. If you want to add a new property, you can add a new line without modifying the previously last line if that line already uses a trailing comma. This makes version-control diffs cleaner and editing code might be less troublesome._

trailing comma가 없으면 객체에 프로퍼티를 추가할 때 기존 라인에 `,`가 붙은 것과 새로 추가된 프로퍼티 줄, 이렇게 총 2줄이 변경되었다고 표시됩니다. trailing comma가 붙어있으면 새로 추가된 프로퍼티 줄만 수정된 것으로 뜨니 수정 사항을 한 눈에 파악하기 좋아요!

```suggestion
  "trailingComma": "all",
```